core/src/main/web/app/helpers/core.js

```diff
@@ -1033,6 +1053,24 @@
             callback(result);
           }
         });
+      },
+      colorize: function (theme) {
+        var colorizedElements = $("html");
+        var ca = colorizedElements.attr('class');
+        var classes = [];
+        if (ca && ca.length && ca.split) {
+          ca = jQuery.trim(ca);
+          /* strip leading and trailing spaces */
+          classes = ca.split(' ');
+        }
+        var themeStylePrefix = "beaker-theme-";
+        var clazz = _.find(classes, function (e) {
```

Comment:
We can try to use undefined 'classes' in this place if remove 'var classes = []'.